packages/TelegramClient-UI.package/TCUChatWindow.class/instance/buttonSendPressed.st

```diff
@@ -1,8 +1,9 @@
 event handling
 buttonSendPressed
 
-	self triggerEvent: #buttonSendPressed with: (Dictionary newFrom: {
-		'chatId' -> self selectedChat id.
-		'message' -> self textInputField contents}).
+	| messageString |
+	
+	messageString := self textInputField contents withBlanksTrimmed.
```

Comment:
maybe do `contents asString` as contents can also be Text.

Reply:
The `withBlanksTrimmed` is defined for both String and Text and when building the Request Json, the message is parsed to string. So everything should be safe, but would you still recommend putting an additional `asString` into here or rename `messageString` to `messageStringOrText` for example?